catalog/pipeline/deploy-release/0.7/README.md

```diff
@@ -12,10 +12,14 @@ Tekton pipeline to verify Snapshot prior to Deployment
 | releasestrategy | The namespaced name (namespace/name) of the releaseStrategy | No | - |
 | snapshot | The namespaced name (namespace/name) of the snapshot | No | - |
 | enterpriseContractPolicy | JSON representation of the policy to be applied when validating the enterprise contract | No | - |
+| enterpriseContractPublicKey | Public key to use for validation by the enterprise contract | Yes | k8s://openshift-pipelines/public-key |
 | verify_ec_task_git_url | The git repo url of the verify-enterprise-contract task | No | - |
 | verify_ec_task_git_revision | The git repo revision the verify-enterprise-contract task | No | - |
 | verify_ec_task_git_pathInRepo | The location of the verify-enterprise-contract task in its repo | No | - |
 
+## Changes since 0.7
```

Comment:
Can you remove this and put L21 in the section below (so move it two lines down)? Same for all of them

Reply:
My apologies. That was a careless mistake. Fixed.